### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).

It also includes the following components:

* [Jamulus/Copyright notices](https://hosted.weblate.org/projects/jamulus/copyright-notices/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Unregistered-Servers](https://hosted.weblate.org/projects/jamulus/unregistered-servers/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: _translator-files/po/nb_NO/Client-Troubleshooting.po [hunk 1]
```diff
@@ -31,7 +32,7 @@ msgstr "wiki"
 #: ../wiki/en/Client-Troubleshooting.md:1
 #, no-wrap
 msgid "/wiki/Client-Troubleshooting"
-msgstr ""
+msgstr "/wiki/klientproblemløsing"
```
Comment (ignotus666) [inline _translator-files/po/nb_NO/Client-Troubleshooting.po]: @comradekingu these links shouldn't be translated.

**Comment by comradekingu** [comment]
@ignotus666 OK. I suppose flagging those as "read-only" in WL helps avoid the mistake.
https://hosted.weblate.org/user/kingu/ is not an admin, so I can't fix that other than to undo the translations.

Edit: The strings are currently just labeled "no-translate" https://hosted.weblate.org/zen/jamulus/navigation/nb_NO/?q=label:%22no-translate%22 which doesn't do a whole lot, whereas for example
https://hosted.weblate.org/translate/jamulus/client-troubleshooting/nb_NO/?checksum=aeeb3af14ed2d239 isn't labeled at all.

**Comment by jujudusud** [comment]
@ignotus666 :
Can you please answer these questions?

https://hosted.weblate.org/translate/jamulus/unregistered-servers/fr/?q=has:comment#comments

https://hosted.weblate.org/translate/jamulus/-/en/?q=has:comment#comments

**Comment by jujudusud** [comment]
French Part of the website translation is ready to be merged.

**Comment by ignotus666** [comment]
@jujudusud answered!

**Comment by jujudusud** [comment]
Thanks a lot @ignotus666

**Comment by henkdegroot** [comment]
All translation for Dutch is included in this PR. Ready to be merged.

**Comment by trebmuh** [comment]
I had time to review and fix the website French translation changes.
It's all good for French now.

File: _translator-files/po/de/Running-a-Server.po [hunk 2]
```diff
@@ -254,9 +255,8 @@ msgstr "2. Unregistriert"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:87
-#, fuzzy
 msgid "This is the default when starting a Server for the first time. Unregistered Servers are not listed by Directories, so only musicians who know your Server's address can connect to it."
-msgstr "Dies ist die Standardeinstellung, wenn du einen Server zum ersten Mal startest. Unregistrierte/private Server werden nicht in Verzeichnissen aufgeführt, so dass nur Musiker, die die Adresse deines Servers kennen, eine Verbindung zum Server herstellen können. Dies ist nützlich, da du in Jamulus nicht kontrollieren kannst, wer sich mit deinem Server verbindet."
```
Comment (ann0see) [inline _translator-files/po/de/Running-a-Server.po:259]: This was correct German.

**Review by ann0see — CHANGES_REQUESTED** [review]

**Review by ann0see — APPROVED** [review]
German is ok too.

**Pull request merged**